ansible_builder/cli.py

```diff
@@ -22,8 +22,11 @@ def run():
     args = parse_args()
     configure_logger(args.verbosity)
 
-    if args.action in ['build']:
-        logger.debug(f'Ansible Builder is building your execution environment image, "{args.tag}".')
+    if args.action in ['create', 'build']:
+        if args.action == 'build':
+            logger.debug(f'Ansible Builder is building your execution environment image, "{args.tag}".')
```

Comment:
I would suggest moving these log statements into the `ansible_builder.main` module inside of the build / create method. This avoids using an `if` and would allow you to order them in a way that makes more sense. We should emit _this_ log until right before running the `podman build` command.